### PR TITLE
More documentation

### PR DESCRIPTION
**Pull request by horationunez**

### CODE REVIEW

**Review by matt-raffel-kiva — APPROVED** [review]

File: portal/docs/overview/overview.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+## Field Partner Testimonial
+
+Kiva piloted the API with Milaap, one of our Field Partners working in India, with very positive results. They reduced the amount of time it takes to post one profile from 10-12 minutes to only 4 minutes. It took Milaap about two weeks to set up and test the API. Here’s what they had to say in their own words: 
+
+> “Setting up the API has significantly reduced the time and effort put into uploading of profiles… This has allowed us to raise more funds in lesser amount of time. In addition to this, internal resources also get freed up and this time can be used for other activities.”
+
+## How do I start using Kiva’s API?
+Kiva’s API is set up as an endpoint that accepts data in a specific format using JSON. To begin using our API, the Field Partner needs to set up their MIS to send information to Kiva’s endpoint in this format.
+
+If you are interested in integrating in 2021, please complete this [form](https://kiva.tfaforms.net/107 "form").
```
Comment (myersadamk) [inline portal/docs/overview/overview.md:29]: We should update this to something like, `If you are interested in integrating in the future`.

I'm interested to hear what Zazil thinks. Maybe this is where the Editorial review stage comes in? Maybe we can add "Target year for integration" to the form?

**Comment by myersadamk** [comment]
Is it possible to remove this link across the pages?

Screen Shot 2022-09-28 at 10 26 02 AM

It takes me to https://github.com/facebook/docusaurus/tree/main/packages/create-docusaurus/templates/shared/docs/overview/structure.md - which is a 404.

File: portal/docs/overview/report_payments.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
```
Comment (myersadamk) [inline portal/docs/overview/report_payments.md:1]: This page looks a lot different in zendesk - it winds up being a bit of text blob without more markup.

**Comment by myersadamk** [comment]
The markdown looks great on many of these sections! It looks like it's largely missing on these pages - are they still in progress?

[Using the API to post draft loans](http://localhost:3000/docs/overview/draftloans)
[Using the API loans fetch endpoint](http://localhost:3000/docs/overview/fetch_loans)
[Using the API to report repayments](http://localhost:3000/docs/overview/report_payments)
[Using the API to post journals](http://localhost:3000/docs/overview/post_journals)

I'm doing a side-by-side with the zendesk site.. does that seem like a reasonable way to review these changes?

**Review by myersadamk — COMMENTED** [review]
Just a couple of notes/questions to follow-up on in a later PR. Overall, it's pretty sweet! 🎸

File: portal/docs/overview/report_payments.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+sidebar_position: 4
+---
+
+# Using the API to report repayments
+
+We recommend that your technical team consults with the Kiva Coordinator at your organization to fully understand Kiva's Repayment Reporting process. To do so, please have your technical team review the following:
+
+This video provides a thorough explanation of the repayment reporting process
```
Comment (myersadamk) [inline portal/docs/overview/report_payments.md:9]: yikes.. the link that is on the Zendesk site points to a private video. I wonder why.. we should try and get an updated link or remove it. Probably an Editoral Stage thing, also.

https://www.youtube.com/watch?v=wPlx30TgV6k

**Pull request merged**